rubric.md

```diff
@@ -0,0 +1,30 @@
+|---------------------------------------------------------------------------------------------------------|--------------------------------|------------------------|--------------------------|
+| Technical Requirements                                                                                  | Does Not Meet Expectations (0) | Meets Expectations (1) | Exceeds Expectations (2) |
+| Project Proposal / Wireframes turned in week 5                                                          |                                |                        |                          |
+| Draft of HTML/CSS turned in week 7                                                                      |                                |                        |                          |
+| First draft of JS turned in week 8                                                                      |                                |                        |                          |
+| Updated rough draft of project week 9                                                                   |                                |                        |                          |
+| Final presentation                                                                                      |                                |                        |                          |
+| Use HTML5 structural elements (header, footer, nav, footer)                                             |                                |                        |                          |
+| Demonstrate a correct use of classes and IDs                                                            |                                |                        |                          |
+| Use classes to identify a type of element and IDs for identifying a specific unique element on the page |                                |                        |                          |
+| Select the appropriate tags to markup content                                                           |                                |                        |                          |
+| Apply fonts, color and styles to elements and the page                                                  |                                |                        |                          |
+| Demonstrate use of floats (or inline-block) and the box model                                           |                                |                        |                          |
+| Use JavaScript/jQuery to make pages interactive                                                         |                                |                        |                          |
+| TOTAL: ______/24                                                                                        |                                |                        |                          |
```

Comment:
Overall great job! Keep up the good work. The grid of images is a trusty framework but don't lean on it too heavily - it can become overwhelming. Also watch the font size and color on your background colors. Looks like you're headed in the right direction so keep it up.

You can merge this pull request whenever you're ready.